soap/client.go

```diff
@@ -44,6 +44,7 @@ type AuthHeader struct {
 type Client struct {
 	URL                    string              // URL of the server
 	Namespace              string              // SOAP Namespace
+	TNamespace			   string              // SOAP Namespace
```

Comment:
What's the difference between Namespace and TNamespace? Comments should clarify this.